sdcm/sct_config.py

```diff
@@ -822,6 +822,11 @@ class SCTConfiguration(dict):
         dict(name="k8s_deploy_monitoring", env="SCT_K8S_DEPLOY_MONITORING", type=boolean,
              help=""),
 
+        dict(name="k8s_local_volume_provisioner_type", env="SCT_K8S_LOCAL_VOLUME_PROVISIONER_TYPE",
+             type=str, choices=("static", "dynamic"),
+             help="Defines the type of the K8S local volume provisioner to be deployed. "
```

Comment:
maybe add reference to the feature?

Reply:
Why?
We use lots of repositories/projects and do not add links to it in configuration...

Reply:
Just missing context what is `static` and `dynamic` in terms of volume provisioning. It is maybe easy for you, but for most people this terms are mystery. Better docs would be helpful.

Reply:
it should at least point to where this is explained ? if it's too much for a configuration option help to explain it